.github/workflows/build.yml

```diff
@@ -1,11 +1,14 @@
 name: build
 
-on: 
+on:
   pull_request:
     branches:
       - main
   workflow_dispatch:
 
+env:
+  ARCHIVE_DIR=/tmp
```

Comment:
@pablofsf : this is invalid syntax and the reason for the broken test pipeline. Please compare the file with latest master. It was already improved to handle pull_requests. Maybe you can simply revert your change. Otherwise please fix this line.

Reply:
Thanks for the pointer, it's fixed now! The archive test also run successfully in the pipeline